shared/editor/embeds/Diagrams.tsx

```diff
@@ -3,43 +3,36 @@ import Frame from "../components/Frame";
 import Image from "../components/Image";
 import { EmbedProps as Props } from ".";
 
-const URL_REGEX = /^https:\/\/viewer\.diagrams\.net\/(?!proxy).*(title=\\w+)?/;
+function Diagrams(props: Props) {
+  const embedUrl = props.attrs.matches[0];
+  const params = new URL(embedUrl).searchParams;
+  const title = params.get("title")
```

Comment:
As it displays below the embed can we show "Draw.io" as title when the embed `settings` is present?

Reply:
Sure!

Reply:
Fixed!